### PR TITLE
2354 Fix "pick displayed columns" spacing issue after supervisor choice

### LINKED ISSUE #2354: Unexpected Gap in Pick Displayed Columns Pop Up
**What type(s) of user does this feature affect?**
 - supervisors
 - admins

**Description**
There is a gap in between Supervisor and Status in the Pick Displayed Columns pop up. Also, it would be nice to alphabetize the options on the Pick Displayed Columns pop up. 

**Screenshots of current behavior, if any**  
Unexpected Gap

_QA Login Details:_  
[Link to QA site](https://casa-qa.herokuapp.com/)  

Login Emails: 
- volunteer1@example.com  view site as a volunteer
- supervisor1@example.com view site as a supervisor
- casa_admin1@example.com view site as an admin

password for all users: 123456

### PR DESCRIPTION
**Pull request by sstacey**
### What github issue is this PR for, if any?
Resolves #2354 

### What changed, and why?
I added a more specific CSS selector to eliminate unintended spacing in the menu. 

### How will this affect user permissions?
- Volunteer permissions: NA
- Supervisor permissions: NA
- Admin permissions: NA

### How is this tested? (please write tests!) 💖💪
Verified in UI.

### Screenshots please :)
Screen Shot 2021-10-01 at 6 49 49 PM

### Feelings gif (optional)
My first ever pull request.  😬
Alt Text

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -414,6 +414,7 @@ GEM
 
 PLATFORMS
   x86_64-darwin-18
+  x86_64-darwin-19
```
Comment (compwron) [inline Gemfile.lock:417]: :)

File: app/javascript/src/stylesheets/shared/layout.scss [hunk 2]
```diff
@@ -1,5 +1,5 @@
 body {
-  background-color: #F4F7FA;
```
Comment (compwron) [inline app/javascript/src/stylesheets/shared/layout.scss:2]: this is an ok change between it doesn't change the actual color :)

**Review by compwron — APPROVED** [review]
Very cool! :)

File: app/javascript/src/stylesheets/shared/layout.scss (hunk 2 above)
Comment (sstacey) [reply app/javascript/src/stylesheets/shared/layout.scss:2]: Ok.  My linter made this change.  I will disable the lint on save moving forward.

**Pull request merged**